workbench-core/base/src/cognitoTokenService.ts

```diff
@@ -27,7 +27,7 @@ export default class CognitoTokenService {
     clientId: string,
     userName: string,
     password: string,
-    accountType: 'USER' | 'ADMIN'
+    accountType: 'USER' | 'ITADMIN'
```

Comment:
I don't think we wanted to pipe this change through `workbench-core`. `ITAdmin` is a SWB concept, not part of MAF.